### PR TITLE
IsaiahB/appeals-42953

### PR DESCRIPTION
**Pull request by IsaiahBar**
Resolves [CorrespondencesAppeal model does not follow naming convention](https://jira.devops.va.gov/browse/APPEALS-42953)

# Description
Change naming covention of CorrespondenceAppeal model

## Acceptance Criteria
- [ ] Code compiles correctly

### CODE REVIEW

File: db/migrate/20231204193100_create_correspondences_appeals.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-class CreateCorrespondencesAppeals < Caseflow::Migration
+class CreateCorrespondenceAppeals < Caseflow::Migration
```
Comment (wmedders21) [inline db/migrate/20231204193100_create_correspondences_appeals.rb:1]: Can't change lines of a previous migration - need to make a new migration

File: db/migrate/20231204195133_add_safe_indices_to_correspondences_appeals.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-class AddSafeIndicesToCorrespondencesAppeals < Caseflow::Migration
+class AddSafeIndicesToCorrespondenceAppeals < Caseflow::Migration
```
Comment (wmedders21) [inline db/migrate/20231204195133_add_safe_indices_to_correspondences_appeals.rb:1]: Can't change lines of a previous migration - need to make a new migration

File: app/models/correspondences_appeal.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class CorrespondencesAppeal < ApplicationRecord
+class CorrespondenceAppeal < ApplicationRecord
```
Comment (wmedders21) [inline app/models/correspondences_appeal.rb:3]: the file name must also be updated to correspondence_appeal.rb

File: spec/models/correspondences_appeal_spec.rb [hunk 4]
```diff
@@ -1,18 +1,18 @@
 # frozen_string_literal: true
 
-RSpec.describe CorrespondencesAppeal, type: :model do
+RSpec.describe CorrespondenceAppeal, type: :model do
```
Comment (wmedders21) [inline spec/models/correspondences_appeal_spec.rb:3]: Update the file name to correspondence_appeal_spec.rb

**Review by wmedders21 — CHANGES_REQUESTED** [review]

File: db/migrate/20240401185955_create_correspondence_appeals.rb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+class CreateCorrespondenceAppeals < Caseflow::Migration
+  def change
+    rename_table :correspondences_appeals, :correspondence_appeals
+  end
+end
+end
```
Comment (HunJerBAH) [inline db/migrate/20240401185955_create_correspondence_appeals.rb:6]: I think this extra `end` is preventing the migration from running and causing the test failures.

**Pull request closed without merging**